### DISCUSSION TITLE
All dates shifted

### DISCUSSION THREAD

**Discussion by Omar8421** [highfestiva/finplot#236] [Q&A]
Hi Highfestiva, I have tried your simple example and noticed that finplot shows all dates shifted one day back (crosshair and x-axis).

import finplot as fplt
import yfinance

df = yfinance.download('QQQ')
df = df.tail(3)
print(df)

fplt.candlestick_ochl(df[['Open', 'Close', 'High', 'Low']])
fplt.show()

I have fixed this problem by doing:     df = df.shift (1, freq = 'D')

Does it only happen on my machine / system?   Or is it a finplot bug?

**Comment by highfestiva** [comment]
finplot defaults to showing datetimes in local timezone. Set your time zone: https://github.com/highfestiva/finplot/wiki/Snippets#time-zone.

**Reply by Omar8421** [reply]
OK!!!   "fplt.display_timezone = datetime.timezone.utc"   work OK for me!
Thank you very much highfestiva!!!
Finplot is excellent!!!
